<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QmyVideoWidget" name="videoWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/001.GIF</normaloff>:/new/prefix1/images/001.GIF</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/622.bmp</normaloff>:/new/prefix1/images/622.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPause">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/620.bmp</normaloff>:/new/prefix1/images/620.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/624.bmp</normaloff>:/new/prefix1/images/624.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>250</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSound">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/volumn.bmp</normaloff>:/new/prefix1/images/volumn.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderVolumn">
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>99</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFullScreen">
         <property name="text">
          <string>全屏</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/237.GIF</normaloff>:/new/prefix1/images/237.GIF</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labCurMedia">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>无曲目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderPosition">
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labRatio">
         <property name="text">
          <string>进度</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmyVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qmyvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
